### ISSUE TITLE
Git-esque tree/branch 

### ISSUE THREAD

**Issue by agritheory** [frappe/charts#180]
Feature request:

![image](https://user-images.githubusercontent.com/18033666/40845164-dbb9a75e-6583-11e8-8a17-8bc2b03d0350.png)

And maybe also vertical? 

![image](https://user-images.githubusercontent.com/18033666/40845217-0a1ec660-6584-11e8-8e74-37a68fae953b.png)

**Comment by scmmishra** [comment]
Hi, at this moment a graph is not on our development roadmap. Can you list any common use cases for this except for github contribution graphs?

**Comment by agritheory** [comment]
Yes. In the ERPNext context it would be handy to visualize a Custom Workflow, represent the manufacturing process for items with multiple BOMs, or showing a process map of the ERPNext docflow. In a more general context, anything that flows.
